docs/reference/accounts.md

```diff
@@ -0,0 +1,168 @@
+# Accounts
+
+TigerBeetle uses the same data structures internally and
+externally. This means that sometimes you need to set temporary values
+for fields that TigerBeetle, not you (the user), are responsible.
+
+### Updates
+
+Account fields *cannot be changed by the user* after
+creation. However, debits and credits fields are updated by
+TigerBeetle as transfers move money to and from an account. And the
+`timestamp` field is set by TigerBeetle after the create request.
+
+## Fields
```

Comment:
If we're going to expose the detail in some might as well expose it in all. I don't think it is less clear for the user. If it means nothing to them they'd gloss over it, I think. And otherwise it helps you to understand IMO

Reply:
:thinking: What I mean is that we are specifying a "type" here, but that may not agree with the type used by their client implementation. e.g. in the reference, we would say that `id` is an unsigned 128-bit integer — but in the javascript client, `id`'s type is a `BigInt`. I think that could be confusing.